### ISSUE TITLE
the Environment variables Issues with node & npm

### ISSUE THREAD

**Issue by kissshot** [coreybutler/nvm-windows#177]
### My Environment
- [ ] Windows 10
- [ ] My Windows installation is non-English.

C:\Users\kissshot>nvm use 4.4.5
Now using node v4.4.5 (64-bit)

nvm is run 

then 
C:\Users\kissshot>npm -v
'npm' 不是内部或外部命令，也不是可运行的程序
或批处理文件。

C:\Users\kissshot>node -v
'node' 不是内部或外部命令，也不是可运行的程序
或批处理文件。

this  mean not found  node  & npm 

my English is not good， hope you can understand

i fixed this ,   delete c:/program files/nodejs

**Comment by millylee** [comment]
+1

**Comment by coreybutler** [comment]
Just saw this... did it work after restarting the terminal?

**Comment by coreybutler** [comment]
I've just released a new version (1.1.1). Can you let me know if this is fixed for you?

**Comment by levilime** [comment]
I was using (1.1.1) and used the installer and the same problem arised on windows 10. Deleting c:/program files/nodejs as @kissshot suggested then opening a new terminal and doing `nvm on` fixed the problem for me.

**Comment by Almusamim** [comment]
Got same issue here.. When i type: nvm use <version> [arch]  I get user control window poping up twice asking if i want to run node 

and it fails when i try it in the command line


```.cmd
C:\Users\Hadi>nvm install 4 64
4.0.0
Downloading node.js version 4.0.0 (64-bit)... Complete
Creating C:\Users\Hadi\AppData\Roaming\nvm\temp

Downloading npm version 2.14.2... Complete
Installing npm v2.14.2...

Installation complete. If you want to use this version, type

nvm use 4.0.0

C:\Users\Hadi>nvm list

    4.0.0

C:\Users\Hadi>nvm use 4 64
4.0.0
Now using node v4.0.0 (64-bit)

C:\Users\Hadi>node -v
'node' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Hadi>npm -v
'npm' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Hadi>nvm use 4 64
4.0.0
Now using node v4.0.0 (64-bit)

C:\Users\Hadi>npm -v
'npm' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Hadi>nvm on
nvm enabled
Now using node v4.0.0 (64-bit)

C:\Users\Hadi>node -v
'node' is not recognized as an internal or external command,
operable program or batch file.

```

**Comment by coreybutler** [comment]
@Almusamim - if you're getting the double prompt, it means you're running with an account that does not have administrative and elevated permissions.

**Comment by robertmirro** [comment]
I seem to be having a similar issue...

I've installed NVM 1.1.1 on Win7 Pro from the GIT 2.9.2 bash shell using "Run as administrator" and then installed the 64-bit versions of node 4.4.7 and latest (6.3.1).  

I then closed that bash shell, opened another one using "Run as administrator" and attempted to use 4.4.7:

```
RMirro ~
$ nvm version
1.1.1

RMirro ~
$ nvm list

    6.3.1
    4.4.7

RMirro ~
$ nvm use 4.4.7
Now using node v4.4.7 (64-bit)

RMirro ~
$ nvm list

    6.3.1
    4.4.7

RMirro ~
$ node -v
bash: node: command not found
```

Any ideas what I'm doing wrong?

**Comment by robertmirro** [comment]
I've resolved this issue but am not certain which step is responsible for my newly found success. :stuck_out_tongue_winking_eye: 

&lt;ugh&gt;

My initial attempt above involved uninstalling node before installing nvm, however I didn't also delete "_C:\Users\ &lt;username&gt; \AppData\Roaming\npm_".  I'm guessing that might be the culprit.

I was able to use [robocopy](http://stackoverflow.com/questions/28175200/unable-to-delete-node-modules-folder/32365809#32365809) to delete that `npm` folder because [rimraf](https://github.com/isaacs/rimraf) wasn't an option (node was uninstalled).

**Comment by coreybutler** [comment]
Win7 isn't officially supported ([read why here](https://github.com/coreybutler/nvm-windows/wiki/Support-for-Windows-7-and-Earlier)), nor is the git bash prompt. Cygwin and anything based off it or it's underlying technologies, or other shell interpretors, typically replace the symlink functionality of the native OS. Another user mentioned success with the latest git bash prompt... I suspect the new version has support for `mklink`, which is the underlying command used in NVM-Windows to create symlinks.... but I haven't explored those shells to find out.

**Comment by hybridview** [comment]
Just a note in case others had my issue or it helps devs troubleshoot. When I ran a command prompt by right-clicking on a folder and choosing "Open Command Prompt", Node and NPM were unavailable and could not be located by using the "where" command.
When I opened a command line by just clicking the shortcut in start menu, Node and NPM could be used and located without issue. Now, I just open a command prompt through start menu shortcut and use "cd" to get to the directory I want. Not ideal, but it doesn't take too much extra effort in my case. I am running Windows Server 2012 R2.
